src/coreclr/jit/compiler.h

```diff
@@ -3023,6 +3024,9 @@ class Compiler
 
     GenTreeIntCon* gtNewIconNode(ssize_t value, var_types type = TYP_INT);
     GenTreeIntCon* gtNewIconNode(unsigned fieldOffset, FieldSeqNode* fieldSeq);
+    GenTreeIntCon* gtNewNull();
+    GenTreeIntCon* gtNewTrue();
+    GenTreeIntCon* gtNewFalse();
```

Comment:
I think it would be nice to preserve the existing convention and call these `gtNew[..]Node`: `gtNewNullNode`, etc.

Reply:
Honestly, I am not a big fan of making it even more verbose, a lot of `gtNew` already don't use that prefix e.g. `gtNewIndir` or `gtNewArrLen` if someone else also support following that convention - I'll rename